go/erasablekv/erasable_kv_store.go

```diff
@@ -284,7 +291,12 @@ func (s *FileErasableKVStore) AllKeys(ctx context.Context) (keys []string, err e
 			continue
 		}
 		parts := strings.Split(file, s.storageDir+string(os.PathSeparator))
-		keys = append(keys, parts[1])
+		key := parts[1]
```

Comment:
Preexisting q: There could be some corner cases where this split does something weird. Is it absolutely guaranteed that `s.storageDir` is absolute? If not, there could be multiple copies of it in a particularly weird path. Could we use `path.Base` to avoid this problem?

Reply:
Also preexisting: Do we need `Glob` above, or could we get away with just `ReadDir`?